app/components/Title/messages.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * HomePage Messages
+ *
+ * This contains all the text for the HomePage component.
+ */
+import { defineMessages } from 'react-intl';
+
+export const scope = 'boilerplate.components.Header';
```

Comment:
it's not big deal however shouldn't we change it to charityActivity.components.Menu ?